src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,66 @@
+/**
+ * D header file to interface with the
+ * $(HTTP pubs.opengroup.org/onlinepubs/9699919799/basedefs/aio.h.html, Linux aio API).
+ * Available since Linux 2.6
+ *
+ * Copyright: Copyright D Language Foundation 2018.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   $(HTTPS github.com/darredevil, Alexandru Razvan Caciulescu)
+ */
+module core.sys.posix.aio;
+
+private import core.sys.posix.signal;
+
+version (CRuntime_Glibc):
```

Comment:
Top of file should be guarded with `version (Posix)`, not glibc.

Reply:
I think the idea is that this has only been tested on linux/Glibc so far, of all Posix platforms.

Reply:
Doesn't mean we should drop the convention used in every other sys/posix module.  New modules that require porting should come with big alarm bells to notify those working on said unsupported platforms to check this code very carefully.

Reply:
Sure, your edit suggestions look good.